<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="str_count">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>302</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>302</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>302</width>
      <height>41</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_back">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>41</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>41</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 120, 6);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;----</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_c">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>41</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>41</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 120, 6);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>ะก</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 6);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 6);

font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 6);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_left">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>232</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 6);
font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_right">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>232</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 6);
font: 18pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>104</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 6);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>168</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 6);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_plus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 160, 6);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_p">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>296</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 120, 6);
font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_eq">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>296</y>
      <width>193</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>193</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>193</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 40, 0);
font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_degree">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>296</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 145, 20);</string>
    </property>
    <property name="text">
     <string>rad=&gt;ยฐ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_tg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>232</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 180, 30);</string>
    </property>
    <property name="text">
     <string>tg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_root">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>168</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(200, 185, 30);</string>
    </property>
    <property name="text">
     <string>โ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_split">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>104</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 160, 6);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>104</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 6);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>104</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 6);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_mult">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>104</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 160, 6);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_0">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>232</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 6);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_minus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 160, 6);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_fact">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>232</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 185, 30);
font: 22pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_8">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>168</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 6);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_st">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>168</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(200, 185, 30);</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>168</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 218, 6);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_pi">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 230, 6);</string>
    </property>
    <property name="text">
     <string>ะฟ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_e">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>104</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 230, 6);
</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_log">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>296</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 180, 30);</string>
    </property>
    <property name="text">
     <string>log10(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_sin">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>232</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 180, 30);</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_cos">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>168</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 180, 30);</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <zorder>str_count</zorder>
   <zorder>b_back</zorder>
   <zorder>b_c</zorder>
   <zorder>b_1</zorder>
   <zorder>b_2</zorder>
   <zorder>b_3</zorder>
   <zorder>b_left</zorder>
   <zorder>b_right</zorder>
   <zorder>b_4</zorder>
   <zorder>b_7</zorder>
   <zorder>b_plus</zorder>
   <zorder>b_p</zorder>
   <zorder>b_eq</zorder>
   <zorder>b_degree</zorder>
   <zorder>b_tg</zorder>
   <zorder>b_root</zorder>
   <zorder>b_split</zorder>
   <zorder>b_5</zorder>
   <zorder>b_6</zorder>
   <zorder>b_mult</zorder>
   <zorder>b_0</zorder>
   <zorder>b_minus</zorder>
   <zorder>b_fact</zorder>
   <zorder>b_8</zorder>
   <zorder>b_st</zorder>
   <zorder>b_9</zorder>
   <zorder>b_e</zorder>
   <zorder>b_log</zorder>
   <zorder>b_pi</zorder>
   <zorder>b_sin</zorder>
   <zorder>b_cos</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
